### ISSUE TITLE
Focus outline does not show for navbar links

### ISSUE THREAD

**Issue by SheepTester** [Objay1003/te-amo-website#6]
This is terrible for accessibility, as those who use the tab key to navigate a website cannot know which link they have selected

It seems your w3.css forces `outline: 0;` on `.w3-bar .w3-bar-item`, so either get rid of it or define your own focus outline.

A focus outline might not be desirable for normal pointer users, so you can use the [`:focus-visible`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible) psuedoclass, which only matches elements focused by keyboard, but this is fairly new, so for older browsers either you need a polyfill or fall back to `:focus` somehow

**Comment by Objay1003** [comment]
Wdym?

**Comment by SheepTester** [comment]
> Wdym?

Some people prefer to use the tab key to navigate websites. On a normal website, there's usually a blue outline around the currently selected button or link. It doesn't show on the navbar links though
